### ISSUE TITLE
libxml2 with icu errors

### ISSUE THREAD

**Issue by crazywhalecc** [crazywhalecc/static-php-cli#94]
When libxml2 depends on icu, other libraries relying on libxml2 will report an error. Caused by 3745dfc931fbb188eb355f64d02a7be17446481f

**Comment by DubbleClick** [comment]
Trying to build with xml (or extensions that depend on xml) doesn't seem to work for me at all:
```
Configuring extensions
checking io.h usability... no
checking io.h presence... no
checking for io.h... no
checking for strtoll... yes
checking for atoll... yes
checking whether to build with LIBXML support... yes
checking for libxml-2.0 >= 2.9.0... no
configure: error: Package requirements (libxml-2.0 >= 2.9.0) were not met:

No package 'libxml-2.0' found

Consider adjusting the PKG_CONFIG_PATH environment variable if you
installed software in a non-standard prefix.

Alternatively, you may set the environment variables LIBXML_CFLAGS
and LIBXML_LIBS to avoid the need to call pkg-config.
See the pkg-config man page for more details.
```

**Comment by DubbleClick** [comment]
Will be fixed by #214
